style.css

```diff
@@ -1,3 +1,5 @@
-#root {
-  color: red;
+* {
```

Comment:
This is usually called a [CSS reset](https://piccalil.li/blog/a-modern-css-reset) in case you weren't aware. It's a good way to reduce browser inconsistencies in things like default line heights, margins and font sizes of headings, etc

Reply:
good to know thank you for showing me